modules/ROOT/pages/7.1.1-release-notes.adoc

```diff
@@ -0,0 +1,145 @@
+= {productname} {release-version}
+:release-version: 7.1.1
+:navtitle: {productname} {release-version}
+:description: Release notes for {productname} {release-version}
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+
+[[overview]]
+== Overview
+
+{productname} {release-version} was released for {enterpriseversion} and {cloudname} on Wednesday, May 22^nd^, 2024.
+
+These release notes provide an overview of the changes for {productname} {release-version}, including:
+
+* xref:accompanying-premium-plugin-changes[Accompanying Premium plugin changes]
+* xref:bug-fixes[Bug fixes]
+
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium plugin changes
+
+The following premium plugin updates were released alongside {productname} {release-version}.
+
+=== PowerPaste
+
+The {productname} {release-version} release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** includes the following fix.
+
+==== A list with the start attribute, separated by `p` tags, resulted in incorrect numbering of lists.
```

Comment:
We decided to move this to 7.2

Reply:
When was the target version changed? Happy to remove if we need it to.

Reply:
@kemister85 Please remove this entry. It is now scheduled to go out with 7.2.